### PR TITLE
EVA-585:Feature/new ebi framework

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,7 +59,8 @@ lib
 temp/*	
 tests/casperjs/*
 report/*
-src/.DS_Store
+.DS_Store/
+src/.DS_Store/
```
Comment (cyenyxe) [inline .gitignore]: In theory these 2 entries [shouldn't be necessary](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository#comment15789759_107921).

File `src/.DS_Store` should be removed from the history: if you check the diff you will see it has been committed.

File: src/css/eva.css [hunk 2]
```diff
@@ -746,20 +655,168 @@ h4{
...
+    color:#064D4D;
+}
+.eva-info-banner i{
+    color:#064D4D;
+}
+
+.eva-info-banner p{
+    margin-left: 0px;
+    font-weight: bold;
+}
```
Comment (cyenyxe) [inline src/css/eva.css]: These 4 `.eva-info-banner` related rules are duplicated.

File: src/css/eva.css [hunk 3]
```diff
@@ -746,20 +655,168 @@ h4{
...
+
+.eva-tiles .content {
+   margin-left:5%;
+   margin-top:2%;
+}
 
+.home-page-section{
+    padding-bottom: 20px;
+}
```
Comment (cyenyxe) [inline src/css/eva.css]: The 3 `.eva-tiles` related and `.home-page-section` rules are duplicated.

File: src/css/eva.css [hunk 4]
```diff
@@ -16,23 +16,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```
Comment (cyenyxe) [inline src/css/eva.css]: This file contains many duplicated rules. I have pointed out some but there are more.

File: src/css/eva.css [hunk 5]
```diff
@@ -746,20 +655,168 @@ h4{
...
+   margin-top:100px;
+}
 
+html.fontface .icon-s2:before{
+    color:#444444;
+}
 
+html.fontface .icon-s2:before{
+    color:#444444;
+}
```
Comment (cyenyxe) [inline src/css/eva.css]: Duplicated rule.

File: src/index.html [hunk 6]
```diff
@@ -1,310 +1,537 @@
...
-<!--<link type="text/css" rel="stylesheet" href="../lib/jsorolla/styles/css/style.css"/>-->
-<link type="text/css" rel="stylesheet" href="css/eva.css"/>
-<!-- /build -->
+<!doctype html>
+<!--
+  ~
+  ~ European Variation Archive (EVA) - Open-access database of all types of genetic
+  ~ variation data from all species
+  ~
+  ~ Copyright 2014, 2015 EMBL - European Bioinformatics Institute
```
Comment (cyenyxe) [inline src/index.html]: Update copyright declaration, the date should be 2014-2017

File: src/index.html [hunk 7]
```diff
@@ -1,310 +1,537 @@
...
+                        <div><p>If you have data in a format other than VCF, which cannot be converted to VCF, please contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>.  Additional submission formats may be supported over time as required by the scientific community.</p></div>
```
Comment (cyenyxe) [inline src/index.html]: This div doesn't seem necessary.

File: src/index.html [hunk 8]
```diff
@@ -1,310 +1,537 @@
...
                     </div>
```
Comment (cyenyxe) [inline src/index.html]: Could this closing tab be indented at the same level than the opening one?

**Comment by kandaj** [comment]
> On 5 Jan 2017, at 15:41, Cristina Yenyxe Gonzalez Garcia <notifications@github.com> wrote:
> 
> @cyenyxe requested changes on this pull request.
> 
> This PR needs to be rebased against develop due to some changes performed at the same time as the support for the new theme. For instance, the "Submit data" page no longer points to "EVA VCF Format" (http://wwwdev.ebi.ac.uk/eva/EVA-585/?VCF <http://wwwdev.ebi.ac.uk/eva/EVA-585/?VCF>).
> 
> I will finish the review after this issue is solved.
> 
 I did rebased against develop branch. must have edited wrong thing while resolving conflicts. all others changes are seems to there. 
> 
> In .gitignore <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > @@ -59,7 +59,8 @@ lib
>  temp/*	
>  tests/casperjs/*
>  report/*
> -src/.DS_Store
> +.DS_Store/
> +src/.DS_Store/
> In theory these 2 entries shouldn't be necessary <https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository#comment15789759_107921>.
> 
> File src/.DS_Store should be removed from the history: if you check the diff you will see it has been committed.
> 
removed.
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > @@ -746,20 +655,168 @@ h4{
>  }
>  
>  
> +.eva-info-banner{
> +    background-color:#94DEDE;
> +    color:#064D4D;
> +}
> +.eva-info-banner span{
> +    color:#064D4D;
> +}
> +.eva-info-banner i{
> +    color:#064D4D;
> +}
> +
> +.eva-info-banner p{
> +    margin-left: 0px;
> +    font-weight: bold;
> +}
> These 4 .eva-info-banner related rules are duplicated.
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> >  
> +.home-page-section{
> +    padding-bottom: 20px;
> +}
> The 3 .eva-tiles related and .home-page-section rules are duplicated.
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > @@ -16,23 +16,19 @@
>   * See the License for the specific language governing permissions and
>   * limitations under the License.
>   */
> +
> This file contains many duplicated rules. I have pointed out some but there are more.
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> >  
> +html.fontface .icon-s2:before{
> +    color:#444444;
> +}
> Duplicated rule.
> 
i removed these css classes as they apply only for the new homepage design. 
> 
> In src/index.html <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > -
> -<link rel="stylesheet" href="../bower_components/bootstrap/dist/css/bootstrap.css">
> -
> -<link rel="stylesheet" href="css/ebi-complaince/ebi-fluid-embl.css">
> -
> -
> -
> -<link type="text/css" rel="stylesheet" href="css/eva.css"/>
> -
> +<!doctype html>
> +<!--
> +  ~
> +  ~ European Variation Archive (EVA) - Open-access database of all types of genetic
> +  ~ variation data from all species
> +  ~
> +  ~ Copyright 2014, 2015 EMBL - European Bioinformatics Institute
> Update copyright declaration, the date should be 2014-2017
> 
updated
> 
> In src/index.html <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > -                        <div><p style="font-size:11px;">This web application makes an intensive use of new web technologies and standards like HTML5. Please see <a href="?About">About</a> for further browser compatibility notes.</p></div>
> +                <div  id="submit" class="hide-div">
> +                    <nav aria-label="You are here:" role="navigation">
> +                        <ul class="breadcrumbs">
> +                            <li><a href="?Home">EVA</a></li>
> +                            <li>
> +                                <span class="show-for-sr">Current: </span> Submit
> +                            </li>
> +                        </ul>
> +                    </nav>
> +                    <h2>Submit</h2>
> +                    <h3>European Variation Archive submissions</h3>
> +                      <div>
> +                        <p>EVA follows the infrastructure of fellow EMBL-EBI resources European Nucleotide Archive (<a href="http://www.ebi.ac.uk/ena/" target="_blank">ENA</a>) and European Genome-phenome Archive (<a href="http://www.ebi.ac.uk/ega/" target="_blank">EGA</a>)  to accept, archive, and accession <a href="?VCF">VCF files</a>. Submissions consist of VCF file(s) and metadata that describe sample(s), experiment(s), and analysis that produced the variant and/or genotype call(s).</p>
> +                        <p>EVA works in collaboration with the Database of Genomics Variants Archive (<a href="http://www.ebi.ac.uk/dgva" target="_blank">DGVa</a>) to accession and archive structural variants. DGVa relies on a template based submission process that is explained in detail <a href="http://www.ebi.ac.uk/dgva/data-submission" target="_blank">here</a></p>                    <p>​Data submitted to EVA is brokered to our collaborating databases at <a href="http://www.ncbi.nlm.nih.gov/" target="_blank">NCBI</a>, <a href="http://www.ncbi.nlm.nih.gov/SNP/" target="_blank">dbSNP</a> and <a href="http://www.ncbi.nlm.nih.gov/dbvar" target="_blank">dbVar</a>. It is therefore  unnecessary to submit data to multiple resources. Please contact <a href="mailto:eva-helpdesk@ebi.ac.uk?subject=Sequence submissionto EVA&amp; target="_top">eva-helpdesk@ebi.ac.uk</a> if you would like any further information on this brokering process or collaboration.</p>
> +                        <div><p>If you have data in a format other than VCF, which cannot be converted to VCF, please contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>.  Additional submission formats may be supported over time as required by the scientific community.</p></div>
> This div doesn't seem necessary.
> 
removed.
> 
> In src/index.html <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15321484>:
> 
> > -                        <div><p style="font-size:11px;">This web application makes an intensive use of new web technologies and standards like HTML5. Please see <a href="?About">About</a> for further browser compatibility notes.</p></div>
> +                <div  id="submit" class="hide-div">
> +                    <nav aria-label="You are here:" role="navigation">
> +                        <ul class="breadcrumbs">
> +                            <li><a href="?Home">EVA</a></li>
> +                            <li>
> +                                <span class="show-for-sr">Current: </span> Submit
> +                            </li>
> +                        </ul>
> +                    </nav>
> +                    <h2>Submit</h2>
> +                    <h3>European Variation Archive submissions</h3>
> +                      <div>
> +                        <p>EVA follows the infrastructure of fellow EMBL-EBI resources European Nucleotide Archive (<a href="http://www.ebi.ac.uk/ena/" target="_blank">ENA</a>) and European Genome-phenome Archive (<a href="http://www.ebi.ac.uk/ega/" target="_blank">EGA</a>)  to accept, archive, and accession <a href="?VCF">VCF files</a>. Submissions consist of VCF file(s) and metadata that describe sample(s), experiment(s), and analysis that produced the variant and/or genotype call(s).</p>
> +                        <p>EVA works in collaboration with the Database of Genomics Variants Archive (<a href="http://www.ebi.ac.uk/dgva" target="_blank">DGVa</a>) to accession and archive structural variants. DGVa relies on a template based submission process that is explained in detail <a href="http://www.ebi.ac.uk/dgva/data-submission" target="_blank">here</a></p>                    <p>​Data submitted to EVA is brokered to our collaborating databases at <a href="http://www.ncbi.nlm.nih.gov/" target="_blank">NCBI</a>, <a href="http://www.ncbi.nlm.nih.gov/SNP/" target="_blank">dbSNP</a> and <a href="http://www.ncbi.nlm.nih.gov/dbvar" target="_blank">dbVar</a>. It is therefore  unnecessary to submit data to multiple resources. Please contact <a href="mailto:eva-helpdesk@ebi.ac.uk?subject=Sequence submissionto EVA&amp; target="_top">eva-helpdesk@ebi.ac.uk</a> if you would like any further information on this brokering process or collaboration.</p>
> +                        <div><p>If you have data in a format other than VCF, which cannot be converted to VCF, please contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>.  Additional submission formats may be supported over time as required by the scientific community.</p></div>
>                      </div>
> Could this closing tab be indented at the same level than the opening one?
> 
done.

File: src/css/eva.css [hunk 9]
```diff
@@ -16,23 +16,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```
Comment (cyenyxe) [inline src/css/eva.css]: Pleas auto-format the file to keep indentation consistent across rules.

File: package.json [hunk 10]
```diff
@@ -25,6 +25,8 @@
     "grunt-contrib-imagemin": "1.0.0",
     "bower": "1.7.9",
     "grunt-bower-task": "~0.4.0",
-    "chromedriver": "2.24.1"
+    "chromedriver": "2.21.2",
```
Comment (cyenyxe) [inline package.json]: Downgrade to chromedriver?

File: src/css/eva.css [hunk 11]
```diff
@@ -16,23 +16,23 @@
...
 }
 
 body.eva {
-    max-width: 1350px;
-    margin: 0 auto;
-    background: #fff url(../img/pale_grey_slight_texture_opt.jpg) top left repeat;
-    /*font-family: 'HelveticaNeueLT Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;*/
+    /*max-width: 1350px;*/
+    /*margin: 50px auto;*/
+    /*background: #fff url(../img/pale_grey_slight_texture_opt.jpg) top left repeat;*/
```
Comment (cyenyxe) [inline src/css/eva.css]: Remove commented code block.

File: src/css/eva.css [hunk 12]
```diff
@@ -93,7 +91,7 @@ div#local-masthead{
 
 .study-browser-panel a:link, .study-browser-panel a:visited,
 .variant-widget-panel a:link, .variant-widget-panel a:visited{
```
Comment (cyenyxe) [inline src/css/eva.css]: Remove rule as it has no content.

File: src/css/eva.css [hunk 13]
```diff
@@ -265,6 +263,14 @@ a, a:active, a:focus{
 .x-panel-header-default-collapsed-left .x-title-text{
```
Comment (cyenyxe) [inline src/css/eva.css]: Remove rule as it has no content.

File: src/css/eva.css [hunk 14]
```diff
@@ -265,6 +263,14 @@ a, a:active, a:focus{
...
+
+.x-tab-bar-default-top > .x-tab-bar-body-default {
+    padding: 5px 0 0 0px;
+}
+
+.x-grid-item-selected {
+     background-color: #ffffff;
+}
 .eva-header-1 {
 /*color: #BF8230;*/
```
Comment (cyenyxe) [inline src/css/eva.css]: Remove commented code (another color set in the next line).

File: src/js/eva.js [hunk 15]
```diff
@@ -473,9 +474,9 @@ Eva.prototype = {
 
         var twitterWidgetEl = document.getElementById('twitter-widget');
         twitterWidgetEl.innerHTML = "";
-        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline" height=100 href="https://twitter.com/EBIvariation"  data-widget-id="437894469380100096">Tweets by @EBIvariation</a>';
+        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline"  href="https://twitter.com/EBIvariation"  height="100" data-widget-id="437894469380100096" data-chrome="noheader nofooter noborders transparent">Tweets by @EBIvariation</a>';
         $.getScript('//platform.twitter.com/widgets.js', function () {
```
Comment (cyenyxe) [inline src/js/eva.js]: Can this be removed or would it prevent the Twitter widget from being displayed?

File: src/js/views/eva-gene-view.js [hunk 16]
```diff
@@ -66,12 +66,12 @@ EvaGeneView.prototype = {
...
         _this.draw(_this.geneData);
 
-        $('#geneViewTabs li').click(function (event) {
-            $(this).toggleClass("active");
-            $(this).siblings().removeClass("active");
-        });
+        // $('#geneViewTabs li').click(function (event) {
+        //     $(this).toggleClass("active");
+        //     $(this).siblings().removeClass("active");
+        // });
```
Comment (cyenyxe) [inline src/js/views/eva-gene-view.js]: Why is this block commented out? Please remove if no longer necessary.

File: src/js/views/eva-gene-view.js [hunk 17]
```diff
@@ -443,21 +443,21 @@ EvaGeneView.prototype = {
...
                 '</div>' +
-                '<div id="scroll-able" class="grid_20">' +
+                '<div class="">' +
+                // '<div class="grid_2" id="geneViewScrollspy">' +
+                // '<span>&nbsp;</span>' +
+                // '<ul id="geneViewTabs" class="nav nav-stacked affix eva-tabs">' +
+                // '<li class="active"><a href="#summary">Summary</a></li>' +
+                // '<li><a href="#transcripts">Variants</a></li>' +
+                // '</ul>' +
+                // '</div>' +
```
Comment (cyenyxe) [inline src/js/views/eva-gene-view.js]: Remove commented code.

File: src/js/views/eva-variant-view.js [hunk 18]
```diff
@@ -84,12 +84,12 @@ EvaVariantView.prototype = {
...
         });
 
-        $('#variantViewTabs li').click(function (event) {
-            $(this).toggleClass("active");
-            $(this).siblings().removeClass("active");
-        });
+        // $('#variantViewTabs li').click(function (event) {
+        //     $(this).toggleClass("active");
+        //     $(this).siblings().removeClass("active");
+        // });
```
Comment (cyenyxe) [inline src/js/views/eva-variant-view.js]: Same as in the gene view file, why is this block commented out? Please remove if no longer necessary.

File: src/js/views/eva-variant-view.js [hunk 19]
```diff
@@ -304,38 +304,38 @@ EvaVariantView.prototype = {
...
-            '</div>' +
-            '<div id="variant-view-scrollable-div" class="col-sm-10 col-md-10 col-lg-10">' +
+            // '<div class="col-sm-1 col-md-1 col-lg-1" id="variantViewScrollspy">' +
+            // '<ul id="variantViewTabs" class="nav nav-stacked affix eva-tabs">' +
+            // '<li class="active"><a href="#summary">Summary</a></li>' +
+            // '<li><a href="#consequenceTypes">Consequence Types</a></li>' +
+            // '<li><a href="#studies">Studies</a></li>' +
+            // '<li><a href="#populationStats">Population Stats</a></li>' +
+            // '</ul>' +
+            // '</div>' +
```
Comment (cyenyxe) [inline src/js/views/eva-variant-view.js]: Remove commented code.

**Comment by kandaj** [comment]
> On 6 Jan 2017, at 13:56, Cristina Yenyxe Gonzalez Garcia <notifications@github.com> wrote:
> 
> @cyenyxe requested changes on this pull request.
> 
> Files src/.DS_Store and vendor/.DS_Store must be removed.
> 
> 
Removed
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -16,23 +16,23 @@
>   * See the License for the specific language governing permissions and
>   * limitations under the License.
>   */
> +
> Pleas auto-format the file to keep indentation consistent across rules.
> 
Formatted
> 
> In package.json <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -25,6 +25,8 @@
>      "grunt-contrib-imagemin": "1.0.0",
>      "bower": "1.7.9",
>      "grunt-bower-task": "~0.4.0",
> -    "chromedriver": "2.24.1"
> +    "chromedriver": "2.21.2",
> Downgrade to chromedriver?
> 
> 
for unknown reasons after my chrome update it was not working with 2.24.1. so i Downgraded to 2.21.2
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> >  @font-face {
>      font-family: 'HelveticaNeueLT Pro';
>      src: url('../fonts/helveticaneueltprolt.otf');
>  }
>  
>  body.eva {
> -    max-width: 1350px;
> -    margin: 0 auto;
> -    background: #fff url(../img/pale_grey_slight_texture_opt.jpg) top left repeat;
> -    /*font-family: 'HelveticaNeueLT Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;*/
> +    /*max-width: 1350px;*/
> +    /*margin: 50px auto;*/
> +    /*background: #fff url(../img/pale_grey_slight_texture_opt.jpg) top left repeat;*/
> Remove commented code block.
> 
> 
done
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -93,7 +91,7 @@ div#local-masthead{
>  
>  .study-browser-panel a:link, .study-browser-panel a:visited,
>  .variant-widget-panel a:link, .variant-widget-panel a:visited{
> Remove rule as it has no content.
> 
> 
done
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -265,6 +263,14 @@ a, a:active, a:focus{
>  .x-panel-header-default-collapsed-left .x-title-text{
> Remove rule as it has no content.
> 
> 
done
> 
> In src/css/eva.css <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -265,6 +263,14 @@ a, a:active, a:focus{
>  .x-panel-header-default-collapsed-left .x-title-text{
>      /*padding-left: 15px;*/
>  }
> +
> +.x-tab-bar-default-top > .x-tab-bar-body-default {
> +    padding: 5px 0 0 0px;
> +}
> +
> +.x-grid-item-selected {
> +     background-color: #ffffff;
> +}
>  .eva-header-1 {
>  /*color: #BF8230;*/
> Remove commented code (another color set in the next line).
> 
> 
done
> 
> In src/js/eva.js <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -473,9 +474,9 @@ Eva.prototype = {
>  
>          var twitterWidgetEl = document.getElementById('twitter-widget');
>          twitterWidgetEl.innerHTML = "";
> -        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline" height=100 href="https://twitter.com/EBIvariation"  data-widget-id="437894469380100096">Tweets by @EBIvariation</a>';
> +        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline"  href="https://twitter.com/EBIvariation"  height="100" data-widget-id="437894469380100096" data-chrome="noheader nofooter noborders transparent">Tweets by @EBIvariation</a>';
>          $.getScript('//platform.twitter.com/widgets.js', function () {
> Can this be removed or would it prevent the Twitter widget from being displayed?
> 
> 
widget will not load if thats removed.
> 
> In src/js/views/eva-gene-view.js <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -66,12 +66,12 @@ EvaGeneView.prototype = {
>          this.targetDiv.innerHTML = _this._varinatViewlayout(_this.geneData);
>          _this.draw(_this.geneData);
>  
> -        $('#geneViewTabs li').click(function (event) {
> -            $(this).toggleClass("active");
> -            $(this).siblings().removeClass("active");
> -        });
> +        // $('#geneViewTabs li').click(function (event) {
> +        //     $(this).toggleClass("active");
> +        //     $(this).siblings().removeClass("active");
> +        // });
> Why is this block commented out? Please remove if no longer necessary.
> 
> 
removed
> 
> In src/js/views/eva-gene-view.js <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> >                  '</div>' +
> -                '<div id="scroll-able" class="grid_20">' +
> +                '<div class="">' +
> +                // '<div class="grid_2" id="geneViewScrollspy">' +
> +                // '<span>&nbsp;</span>' +
> +                // '<ul id="geneViewTabs" class="nav nav-stacked affix eva-tabs">' +
> +                // '<li class="active"><a href="#summary">Summary</a></li>' +
> +                // '<li><a href="#transcripts">Variants</a></li>' +
> +                // '</ul>' +
> +                // '</div>' +
> Remove commented code.
> 
> 
done
> 
> In src/js/views/eva-variant-view.js <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > @@ -84,12 +84,12 @@ EvaVariantView.prototype = {
>              }
>          });
>  
> -        $('#variantViewTabs li').click(function (event) {
> -            $(this).toggleClass("active");
> -            $(this).siblings().removeClass("active");
> -        });
> +        // $('#variantViewTabs li').click(function (event) {
> +        //     $(this).toggleClass("active");
> +        //     $(this).siblings().removeClass("active");
> +        // });
> Same as in the gene view file, why is this block commented out? Please remove if no longer necessary.
> 

done.
> 
> In src/js/views/eva-variant-view.js <https://github.com/EBIvariation/eva-web/pull/30#pullrequestreview-15487154>:
> 
> > -            '<ul id="variantViewTabs" class="nav nav-stacked affix eva-tabs">' +
> -            '<li class="active"><a href="#summary">Summary</a></li>' +
> -            '<li><a href="#consequenceTypes">Consequence Types</a></li>' +
> -            '<li><a href="#studies">Studies</a></li>' +
> -            '<li><a href="#populationStats">Population Stats</a></li>' +
> -            '</ul>' +
> -            '</div>' +
> -            '<div id="variant-view-scrollable-div" class="col-sm-10 col-md-10 col-lg-10">' +
> +            // '<div class="col-sm-1 col-md-1 col-lg-1" id="variantViewScrollspy">' +
> +            // '<ul id="variantViewTabs" class="nav nav-stacked affix eva-tabs">' +
> +            // '<li class="active"><a href="#summary">Summary</a></li>' +
> +            // '<li><a href="#consequenceTypes">Consequence Types</a></li>' +
> +            // '<li><a href="#studies">Studies</a></li>' +
> +            // '<li><a href="#populationStats">Population Stats</a></li>' +
> +            // '</ul>' +
> +            // '</div>' +
> Remove commented code.
> 

done.

**Pull request merged**